src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/sdl_activate_app_request.cc

```diff
@@ -186,6 +186,13 @@ void SDLActivateAppRequest::Run() {
 
   if (app_to_activate->IsRegistered()) {
     SDL_LOG_DEBUG("Application is registered. Activating.");
+    auto main_state = app_to_activate->CurrentHmiState(
+        mobile_apis::PredefinedWindows::DEFAULT_WINDOW);
+    if (mobile_apis::HMILevel::INVALID_ENUM == main_state->hmi_level()) {
+      SDL_LOG_DEBUG("Application not in NONE, postpone activation");
```

Comment:
Please consider rewording of this debug to more clear message like "Application registration is not completed, HMI level hasn't set yet, postpone activation" or something like that

Reply:
@ychernysheva please check the latest updates